dev/run-tests.py

```diff
@@ -253,9 +253,14 @@ def kill_zinc_on_port(zinc_port):
     """
     Kill the Zinc process running on the given port, if one exists.
     """
-    cmd = ("/usr/sbin/lsof -P |grep %s | grep LISTEN "
-           "| awk '{ print $2; }' | xargs kill") % zinc_port
-    subprocess.check_call(cmd, shell=True)
+    try:
+        cmd = ("lsof -P |grep %s | grep LISTEN "
+               "| awk '{ print $2; }' | xargs kill") % zinc_port
+        subprocess.check_call(cmd, shell=True)
+    except:
```

Comment:
Could we catch the explicit exception?

Also, I think we could this like:

```python
cmd = "%s -P |grep %s | grep LISTEN | awk '{ print $2; }' | xargs kill"
...
lsof = "lsof"
subprocess.check_call(cmd % (lsof  zinc_port), shell=True)
...
lsof = "/usr/sbin/lsof"
subprocess.check_call(cmd % (lsof  zinc_port), shell=True)
```

Reply:
Yes, if the command does not exist, an exception occurs. Thus, we can execute one of the two cases.

Yea, to use `cmd` is fine.